### PR TITLE
Add `fetchOptions` configuration object

### PR DESCRIPTION
**Pull request by ArnaudWeyts**
### Added

- Added `fetchOptions` object to global and local configuration

### Removed

- [BREAKING] `additionalHeaders` configuration object, use the new `fetchOptions` configuration with a `headers` property instead

### CODE REVIEW

**Review by mrtus — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]

**Review by sanderbrugge — APPROVED** [review]
👌 
pinging @JorenSaey just so you're aware, since you implemented `additionalHeaders` initially.

**Comment by JorenSaeyTL** [comment]
Ah yeah nice! Will need to be adapted for the new companies overview

File: README.md [hunk 1]
```diff
@@ -84,27 +84,31 @@ const init = async () => {
...
 ```js
 import API from '@teamleader/api';
 
 const { users } = API({
   getAccessToken: () => 'thisisatoken',
-  additionalHeaders: {
-    'x-tl-feature-flags': 'core.some-feature-flag=true',
+  fetchOptions: {
+    headers: {
+      'x-tl-feature-flags': 'core.some-feature-flag=true',
```
Comment (JorenSaeyTL) [inline README.md:98]: Could be a good oportunity to use a different example header, as we dont want to give away that integrators can do this?

**Comment by ArnaudWeyts** [comment]
> Ah yeah nice! Will need to be adapted for the new companies overview

@JorenSaeyTL, I'll be creating an upgrade PR with the needed changes

File: README.md [hunk 2]
```diff
@@ -84,27 +84,31 @@ const init = async () => {
...
 });
 
 const init = async () => {
   // (options, plugins)
   const me = await users.me(undefined, {
-    additionalHeaders: {
-      'x-tl-feature-flags': 'core.some-feature-flag=true',
+    fetchOptions: {
+      headers: {
+        'x-tl-feature-flags': 'core.some-other-feature-flag=true',
```
Comment (JorenSaeyTL) [inline README.md:108]: Same here

**Review by JorenSaeyTL — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ArnaudWeyts) [reply README.md:98]: Hmm, yeah but then I'd like to pick it up in a bigger documentation overhaul initiative

**Pull request merged**